aggregator/src/main/resources/application.properties

```diff
@@ -4,22 +4,34 @@ quarkus.http.port=8086
 %test.quarkus.http.port=9086
 %test.quarkus.http.test-port=9086
 
+## configure your datasource
+#quarkus.datasource.db-kind=postgresql
+#quarkus.datasource.username=postgres
+#quarkus.datasource.password=postgres
+#quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
 # configure your datasource
 quarkus.datasource.db-kind=postgresql
 quarkus.datasource.username=postgres
-quarkus.datasource.password=postgres
-quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
+quarkus.datasource.password=mysecretpassword
+quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/postgres
+quarkus.datasource.jdbc.driver=org.postgresql.Driver
+quarkus.hibernate-orm.dialect=org.hibernate.dialect.PostgreSQL82Dialect
+#quarkus.hibernate-orm.database.generation=drop-and-create
 
 quarkus.hibernate-orm.physical-naming-strategy=com.redhat.cloud.notifications.db.naming.SnakeCasePhysicalNamingStrategy
 
+quarkus.aggregator.serializer=org.apache.kafka.common.serialization.StringSerializer
+quarkus.aggregator.broker-url=localhost:9092
+quarkus.aggregator.acks=1
+quarkus.aggregator.topic.name=platform.notifications.aggregation
+
```

Comment:
I assume the 4 variables above are ours and not quarkus ones ?
So I guess they shouldn't start with `quarkus.` but rather `notifications.`, no ?

Reply:
Good catch. Thanks :) 
Fixed with the last PR.